web/react/components/post_body.jsx

```diff
@@ -126,7 +126,7 @@ module.exports = React.createClass({
                 } else if (i < Constants.MAX_DISPLAY_FILES) {
                     postFiles.push(
                         <div className="post-image__column custom-file" key={fileInfo.name+i}>
-                            <a href={fileInfo.path+"."+fileInfo.ext} download={fileInfo.name+"."+fileInfo.ext}>
+                            <a href={fileInfo.path + (fileInfo.ext ? "." + fileInfo.ext : "")} download={fileInfo.name + (fileInfo.ext ? "." + fileInfo.ext : "")}>
```

Comment:
I'll keep that in mind for later. I'm going to leave this as it is though since the updated file UI that I'm working on will be removing this code anyway.